python/az/aro/azext_aro/_params.py

```diff
@@ -54,10 +57,25 @@ def load_arguments(self, _):
         c.argument('service_cidr',
                    help='CIDR of service network. Must be a minimum of /18 or larger.',
                    validator=validate_cidr('service_cidr'))
+        c.argument('software_defined_network', arg_type=get_enum_type(['OVNKubernetes', 'OpenShiftSDN']),
+                   options_list=['--sdn-type'],
+                   help='SDN type either "OVNKubernetes" or "OpenShiftSDN (default)"',
+                   validator=validate_sdn)
 
+        c.argument('disk_encryption_set',
+                   help='ResourceID of the DiskEncryptionSet to be used for master and worker VMs.',
+                   validator=validate_disk_encryption_set)
+        c.argument('master_encryption_at_host', arg_type=get_enum_type(['Enabled', 'Disabled']),
+                   options_list=['--master-enc-at-host'],
```

Comment:
is there reason we use `--master-enc-at-host` and not `--master-encryption-at-host`? Matches original flag name

Reply:
I had master-encryption-at-host, but that doesn't pass validation since the parameter name is too long.

Reply:
Ok, found the linter rule, and at least one version of the parameter has to be <22 characters (not counting -- or - in front). I'm wondering if I should shorten the short versions even more.

Reply:
To sum up:
- added long names and shortened the short names a bit
- changed the param type to tri-state (so that you don't have to do "Enabled")
- removed param validator because it's not needed.